.github/scripts/compile.sh

```diff
@@ -35,13 +35,37 @@ do_patch () {
 
 WORKING=`pwd`
 wget ${PROTON_SOURCE_URL} -O qpid-proton.tar.gz
+wget ${LWS_SOURCE_URL} -O libwebsockets.tar.gz
 
-mkdir -p qpid-proton-src build staging proton_build proton_install
+mkdir -p qpid-proton-src build staging proton_build proton_install lws-src lws_build lws_install
 tar -zxf qpid-proton.tar.gz -C qpid-proton-src --strip-components 1
+tar -zxf libwebsockets.tar.gz -C lws-src --strip-components 1
 
 do_patch "patches/proton" qpid-proton-src
 
-cd proton_build
+cd $WORKING/lws_build
+cmake \
+    -D LWS_LINK_TESTAPPS_DYNAMIC=ON \
```

Comment:
This is minor but just for consistency sake let's remove the space between the -D and the options.  
E.g. instead of `-D LWS_WITH_LIBUV=OFF`, let's do `-DLWS_WITH_LIBUV=OFF`

Reply:
Is it more valuable to be consistent with libwebsockets.spec file or with other cmake builds in this dockerfile? :P

Reply:
I would like it if it were consistent with other builds in this file.

Reply:
/me quietly withdraws from the thread

Reply:
agree that we should make it consistent with other formats in the project